pkg/components/clusterversionoperator/component.go

```diff
@@ -50,6 +50,18 @@ var ClusterVersionOperatorComponent = Component{
 					"[sig-cluster-lifecycle] TestAdminAck should succeed [Suite:openshift/conformance/parallel]",
 				},
 			},
+			{Suite: "Operators related features"},
+			{Suite: "Cluster_Operator"},
+			{Suite: "Display All Namespace Operands for Global Operators"},
+			{Suite: "OTA"},
+			{Suite: "Operators related features"},
+			{Suite: "Scenarios which will be used both for function checking and upgrade checking"},
+			{Suite: "basic verification for upgrade testing"},
+			{Suite: "cluster upgrade"},
+			{Suite: "fips enabled verification for upgrade"},
+			{Suite: "operand tests"},
+			{Suite: "Operators Installed nonlatest operator test"},
+			{Suite: "Operators related features on sts cluster mode"},
```

Comment:
I think only `OTA` and `cluster upgrade` need to map to CVO.

Reply:
Actually I think it will be easier to understand that mapping  `CVO` and `cluster upgrade` to `OTA`. To achieve that, sounds we need update golang e2e test case keyword.

Reply:
Currently we are using `OTA cvo` and `OTA osus` for the golang e2e test and `cluster upgrade` for the upgrade test. Maybe we can map `OTA` and `cluster upgrade` to `OTA`? cc @jiajliu for discussion

Reply:
> Maybe we can map OTA and cluster upgrade to OTA

That sounds good to me.

Reply:
Mapping to `OTA` instead of current `clusterversionoperator` sounds reasonable. It looks like sub directory could be created on component dir, maybe we could use sub dirs under `OTA` component for a better mapping.
```
$ tree ota
ota
├── cvo
├── oc
├── osus
└── upgrade
```
then the ginkgo cases and upgrade could be mapping to each sub directory clearly.